__tests__/app.test.js

```diff
@@ -43,5 +43,45 @@ describe('App', ()=>{
             })
         })
     })
+    describe('GET /api', () => {
+        test('GET 200: should provide a description, accepted queries, format of the request body and example response of all other endpoints available in an object', () => {
```

Comment:
This is absolutely fine, but just a heads up in case you want to refactor this. You could instead require in the JSON file and the in your expect, compare the body to the actual json file. This would significantly reduce you test code for this end point